extensions/wikia/CuratedContent/CuratedContentHooks.class.php

```diff
@@ -71,4 +71,16 @@ public static function onOutputPageBeforeHTML() {
 		}
 		return true;
 	}
+
+	public static function onBeforePrepareActionButtons( $actionButton, &$contentActions ) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "$actionButton". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)